.vscode/settings.json

```diff
@@ -1,8 +1,8 @@
 {
-    "editor.formatOnSave": true,
-    "rust-analyzer.rustfmt.enableRangeFormatting": false,
-    "[rust]": {
-        "editor.defaultFormatter": "matklad.rust-analyzer"
-    },
-    "files.insertFinalNewline": true
+  "editor.formatOnSave": true,
+  "rust-analyzer.rustfmt.enableRangeFormatting": false,
+  "[rust]": {
+    "editor.defaultFormatter": "rust-lang.rust-analyzer"
```

Comment:
Oh I think you might have been using a non-official Rust Analyzer plugin

This one is better!